[2023-11-06T12:06:03.233+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-17T00:00:00+00:00 [queued]>
[2023-11-06T12:06:03.251+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-17T00:00:00+00:00 [queued]>
[2023-11-06T12:06:03.252+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:06:03.277+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-10-17 00:00:00+00:00
[2023-11-06T12:06:03.283+0000] {standard_task_runner.py:57} INFO - Started process 843 to run task
[2023-11-06T12:06:03.289+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-10-17T00:00:00+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmpt213lpeo']
[2023-11-06T12:06:03.294+0000] {standard_task_runner.py:85} INFO - Job 91: Subtask task_a
[2023-11-06T12:06:03.392+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-17T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:06:03.555+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-10-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-17T00:00:00+00:00'
[2023-11-06T12:06:03.557+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:06:03.559+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World :)']
[2023-11-06T12:06:03.577+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:06:03.579+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: syntax error near unexpected token `)'
[2023-11-06T12:06:03.580+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: `echo Hello World :)'
[2023-11-06T12:06:03.581+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-11-06T12:06:03.611+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-11-06T12:06:03.619+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231017T000000, start_date=20231106T120603, end_date=20231106T120603
[2023-11-06T12:06:03.649+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 91 for task task_a (Bash command failed. The command returned a non-zero exit code 2.; 843)
[2023-11-06T12:06:03.670+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-06T12:06:03.784+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-06T12:19:06.217+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-17T00:00:00+00:00 [queued]>
[2023-11-06T12:19:06.237+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-17T00:00:00+00:00 [queued]>
[2023-11-06T12:19:06.238+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:19:06.277+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-10-17 00:00:00+00:00
[2023-11-06T12:19:06.286+0000] {standard_task_runner.py:57} INFO - Started process 2041 to run task
[2023-11-06T12:19:06.294+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-10-17T00:00:00+00:00', '--job-id', '163', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmpd8nu8q4j']
[2023-11-06T12:19:06.299+0000] {standard_task_runner.py:85} INFO - Job 163: Subtask task_a
[2023-11-06T12:19:06.409+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-10-17T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:19:06.622+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-10-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-17T00:00:00+00:00'
[2023-11-06T12:19:06.625+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:19:06.627+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World.']
[2023-11-06T12:19:06.654+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:19:06.662+0000] {subprocess.py:93} INFO - Hello World.
[2023-11-06T12:19:06.666+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-11-06T12:19:06.764+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231017T000000, start_date=20231106T121906, end_date=20231106T121906
[2023-11-06T12:19:06.925+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-06T12:19:06.973+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
